PermissionsEx/permissions.yml

```diff
@@ -8383,6 +8383,34 @@ groups:
         - ch.alias.haste2
         - ch.alias.haste3
         - ch.alias.haste4
+  delsinrowe:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&4D&0e&4l&cs&0i&cn &8R&7o&8w&7e'
+      suffix: '&4'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.delsinrowe
+        - ch.alias.jump0
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.intangible
+        - ch.alias.tangible
```

Comment:
denied, he wasnt going through walls, he was in a smoke form that passed through a grate.